Makefile.in

```diff
@@ -14,7 +14,9 @@ CPU = @target_cpu@
 TARGET = @target_alias@
 SYMBOL_PREFIX = @SYMBOL_PREFIX@
 CFLAGS = @CFLAGS@
-INCLUDES = -I$(srcdir) @bfd_include_dir@ @binutils_include_dir@ @zlib_include_dir@
+INCLUDES = -I$(srcdir) -Ibfd-headers @zlib_include_dir@
+BFD_INCLUDE_DIR = $(shell cd "@bfd_include_dir@" && pwd)
```

Comment:
my suggestion for doing it in configure means you don't have to do it in the makefile.  `$(shell ...)` isn't exactly portable, and it'd be executed by every line that tries to expand the vars.  prob doesn't happen often now, but it's considered bad form.